src/ContainerBuilder.php

```diff
@@ -150,10 +150,8 @@ public function build()
         // Mutable definition source
         $source->setMutableDefinitionSource(new DefinitionArray([], $autowiring));
 
-        if ($this->sourceCache) {
-            if (!SourceCache::isSupported()) {
-                throw new \Exception('APCu is not enabled, PHP-DI cannot use it as a cache');
-            }
+        // use cache if isSupported check passes, otherwise proceed without cache and do not throw an exception
```

Comment:
I like this change but if there was an exception before then this change may break BC because someone may expect this exception for some cases... 
I mean that exception was throwing when someone was trying to use APCu cache without enabled php extension. The new version just skipping the cache layer. 
Maybe we should notify users somehow about that:
```php
trigger_error("You're trying to use APCu that is not available", E_USER_WARNING);
```
Of course adding such message still will break BC

Reply:
yeah, since this is a BC I see small changes for it to be merged into original repo. But still it might be more convenient to just leave `sourceCache` turned on and expect the things to work, even if apcu breaks - the project should not crash in such case - and it should not be required to wrap $container->build() with try...catch in every implementation